lib/pbench/server/hierarchy.py

```diff
@@ -62,7 +62,7 @@ class Hierarchy(object):
     BAD_PREFIXES = "bad_prefixes"
     UNEXPECTED_USER_LINKS = "unexpected_user_links"
     WRONG_USER_LINKS = "wrong_user_links"
-    IMPOSSIBLE = "impossible"
+    INCONCEIVABLE = "inconceivable"
```

Comment:
This will mean nothing to you, but it has to be said, so I'll get it over with. (Famous Princess Bride quote):

> "Inconceivable!"
>
> "You keep using that word. I do not think it means what you think it means."

With that out of the way, I like it better than "impossible" and it suggests that what happened is something we arrogantly expected not to happen rather than something that can't happen. And if someone reading the code hears an echo of Inigo's response to Vezzini ... well, that never hurt anyone. ;-)

Reply:
I can understand what you are trying to say :+1: